projects/max22007/_builds.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
this should be `builds.json`. Remove the underscore `_` from the filename. Otherwise it won't be parsed by the CI. Unless you have the same issue as here: https://github.com/analogdevicesinc/no-OS/commit/df789f67eab6ff4be614dc8a5f6ae31a2d1aaf9d

Reply:
Renamed the file